### ISSUE TITLE
[IBM Power Up] Your Project Title

### ISSUE THREAD

**Issue by Yiningxie** [XanaduAI/QHack2022#44]
### Team Name: QH

*Your team's name (matching the name used on the QHack Coding Challenges, if applicable)*

### Project Description: Analyzing Chinese Medicine using Quantum Computers in Fighting Pandemics

*A brief description of your project (1-2 paragraphs).*

Chinese Medicine has been used successfully in China to fight COVID-19 pandemic, and Chinese herbs are used together at the same time in treating diseases, including in fighting recent COVID-19. This project is to use currently available quantum computers to analyze various components in those herbs and their effects so that we’ll know what specific chemical substances/drugs in these herbs that we could use in fighting next pandemics of those viruses and their variants widely.

After my honorable graduation from Tongji Medical School at Huazhong University of Science and Technology where I learned Western Medicine and some Chinese Medicine, I did my Residency on Internal Medicine at Union (Xie He) Hospital.  I then went to Germany and did my doctoral thesis at Munich University. After obtaining my doctoral degree with honor from Munich University, I came to University of California at Los Angeles (UCLA) for my post-doctoral fellowship and subsequently worked there. I also took and successfully passed the U.S. National Board Step-1, National Board Step-2 and Clinical Skill Assessment (CSA), and am certified by the U.S. Educational Commission for Foreign Medical Graduates (ECFMG).

In 2006, I moved from Los Angeles (UCLA) to Washington, DC for my job at the National Institutes at Health (NIH).  My grandfather from my mother side had been a very good doctor practicing Chinese Medicine in Wuhan before he passed.  My mother had been an excellent pediatrician who treated her patients with Western Medicine together with Chinese Medicine that she learned from her father.  From this historical and family influence, I paid very close attention to how Chinese Medicine was successfully used in fighting COVID-19 in China since the outbreak in Wuhan, while I’ve been following up with COVID-19 in U.S.

Starting April 2020, I’ve been using Chinese Medicine on myself in fighting the challenges.  My e-mail is [yiningxie3@gmail.com](mailto:yiningxie3@gmail.com) and my phone # is 240-453-1534.  I look forward to hearing from you about my draft of this QHack Open Hackathon project.

Many thanks, Sincerely,

Yining Xie

### Source code: 

The Quantum Programming Language Twist from MIT will be used in conjunction with PenneLane (Pythone, Numby, PyTorch, TensorFlow, JAX interface, Cirq, Strawberrybields, Qiskitt, Forest, etc)

https://doi.org/10.1145/3498691

Two Attachments here: 3498691.pdf; appendices.pdf 
[3498691.pdf](https://github.com/XanaduAI/QHack/files/8121799/3498691.pdf)
[appendices.pdf](https://github.com/XanaduAI/QHack/files/8121801/appendices.pdf)

*A hyperlink to the draft source code for your team's hackathon project (e.g., a GitHub repo).*

### Resource Estimate: 

If awarded, the access to IBM Quantum machine with IBM 16-qubit QPU will be used to finish this project.

The Quantum Programming Language Twist from MIT will be used in conjunction with PenneLane (Pythone, Numby, PyTorch, TensorFlow, JAX interface, Cirq, Strawberrybields, Qiskitt, Forest, etc).

*A 1-2 paragraph written Resource Estimate, indicating how you expect to use the IBM 16-qubit QPU, if awarded, to finish your Open Hackathon project.*

**Comment by isaacdevlugt** [comment]
Thank you for your Power Up submission! As a reminder, the final deadline for your project is February 25 at 17h00 EST. Submissions should be done here: https://github.com/XanaduAI/QHack/issues/new?assignees=&labels=&template=open_hackathon.md&title=%5BENTRY%5D+Your+Project+Title

This issue will be closed shortly.

Good luck!
